theme-data/win-system-hc/win-system-hc-common.json

```diff
@@ -143,6 +143,17 @@
           "attention": "@color-hc-PlaceholderColor",
           "secure": "@color-hc-PlaceholderColor"
         }
+      },
+      "content": {
+        "content-pdf": "@color-partner-pdf",
```

Comment:
You get the values from here. In this case, all content colors would be @color-hc-SystemColorHighlightColor

"pdf": "@color-hc-SystemColorHighlightColor"
"word": "@color-hc-SystemColorHighlightColor"
"powerpoint": "@color-hc-SystemColorHighlightColor"
...
"keynote": "@color-hc-SystemColorHighlightColor"